### PR TITLE
fix(calendar-utils): `getPaddingDayCount` helper function

### PR DESCRIPTION
**Pull request by kark**
#### Summary
Closes https://github.com/commercetools/ui-kit/issues/2632
Fixes a bug introduced [here](https://github.com/commercetools/ui-kit/pull/2598)

### CODE REVIEW

File: packages/calendar-utils/src/calendar.ts [hunk 1]
```diff
@@ -21,7 +20,7 @@ export const getPaddingDayCount = (day: MomentInput, locale: string) => {
   const paddingDayCount = (firstDayOfMonth - firstDayOfWeek + 7) % 7;
 
   // ensure number is always positive
```
Comment (CarlosCortizasCT) [inline packages/calendar-utils/src/calendar.ts:22]: **nit**: now this message is a bit misleading

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks 👍

File: packages/calendar-utils/src/calendar.ts (hunk 1 above)
Comment (kark) [reply packages/calendar-utils/src/calendar.ts:22]: Thank you for pointing it out, I think it's still true - we always return a number in range 0-6.

Comment (CarlosCortizasCT) [reply packages/calendar-utils/src/calendar.ts:22]: Yes, you're right. 
What I was trying to highlight is that the comment, which refers to the next line, does not match the next instruction: we previously made sure that assumption with the call to `inRange(paddingDayCount, 0, 6)` but the the next call just makes sure `paddingDayCount` is a number, it does not validate what type of number is.

Anyway, this is just a tiny recommendation from my part.

Comment (kark) [reply packages/calendar-utils/src/calendar.ts:22]: Ah, I understand now. Thanks for the explanation; it makes perfect sense to me!

**Comment by kark** [comment]
I'd like to bring to your attention that I couldn't find any explanation of the change in visual regression tests for the `<Tag>` component. This is consistent across all new branches, and I haven't noticed any recent alterations in the baseline. AFAIK, the current modification appears to be correct. I'm just raising this for awareness.

**Pull request merged**